src/js/order.js

```diff
@@ -685,7 +687,7 @@ function fetchYear(year, request_scheduler, nocache_top_level) {
             '&returnTo=' +
             '&orderFilter=year-%(year)s' +
             '&startIndex=%(startOrderPos)s' +
-            '&language=en_GB'],
+            '&language=es_ES'],
```

Comment:
Unless you've tested this with a real spanish account, we shouldn't change it: we know that it was working when it was first set up. Remember that the user never sees the english or spanish responses - they are scraped by logic that is probably expecting english patterns and so this change may break the scraping.
If we had (yeah, I know I am a bore) json dumps for example normal and digital orders, then we could be more bold, but I think we should leave this alone unless we have an actively collaborating ES user.